pkgs/by-name/cs/csharpier/package.nix

```
@@ -0,0 +1,18 @@
+{ buildDotnetGlobalTool, lib }:
+
+buildDotnetGlobalTool {
+  pname = "csharpier";
+  version = "0.26.4";
+  executables = "dotnet-csharpier";
+
+  nugetSha256 = "sha256-9ieSAWulfJZz0INRf95Nz5C865sM8kffK2UGZ1NZ6Z0=";
+
+  meta = with lib; {
+    description = "CSharpier is an opinionated code formatter for c#. ";
+    homepage = "https://csharpier.com/";
+    changelog = "https://github.com/belav/csharpier/blob/main/CHANGELOG.md";
+    license = licenses.free;
```

[Upstream seems to specify MIT license](https://github.com/belav/csharpier/blob/main/LICENSE)

```suggestion
    license = licenses.mit;
```